src/SFA.DAS.EmployerUsers.Api.Client/EmployerUsersApiClient.cs

```diff
@@ -37,7 +37,7 @@ public EmployerUsersApiClient(EmployerUsersApiConfiguration configuration) : thi
 
         public async Task<PagedApiResponseViewModel<UserSummaryViewModel>> SearchEmployerUsers(string criteria, int pageNumber = 1, int pageSize = 1000)
         {
-            return await GetResource<PagedApiResponseViewModel<UserSummaryViewModel>>($"/api/users/search/{criteria}?pageNumber={pageNumber}&pageSize={pageSize}");
+            return await GetResource<PagedApiResponseViewModel<UserSummaryViewModel>>($"/api/users/search?criteria={criteria}&pageNumber={pageNumber}&pageSize={pageSize}");
```

Comment:
The following should work instead - the change you've done here is missing the change in the API as well

```c#
return await GetResource<PagedApiResponseViewModel<UserSummaryViewModel>>($"/api/users/search/{criteria}/?pageNumber={pageNumber}&pageSize={pageSize}");
```